external/textprogressbar/textprogressbar.m

```diff
@@ -40,29 +42,37 @@ function textprogressbar(c)
     fprintf([c '\n']);
 elseif isnumeric(c)
     % Progress bar - normal progress
-    c = floor(c);
-    percentageOut = [num2str(c) '%%'];
+    WAITBAR_PROGRESS = floor(c);    
+    percentageOut = [num2str(WAITBAR_PROGRESS) '%%'];
     percentageOut = [percentageOut repmat(' ',1,strPercentageLength-length(percentageOut)-1)];
-    nDots = floor(c/100*strDotsMaximum);
+    nDots = floor(WAITBAR_PROGRESS/100*strDotsMaximum);
     dotOut = ['[' repmat('.',1,nDots) repmat(' ',1,strDotsMaximum-nDots) ']'];
-    strOut = [percentageOut dotOut];
+    WAITBAR_STRING = [percentageOut dotOut];
 
     % Print it on the screen
     if WAITBAR_HANDLE == -1
         % Don't do carriage return during first run
-        fprintf(strOut);
+        fprintf(WAITBAR_STRING);
     else
         % Do it during all the other runs
-        fprintf([WAITBAR_HANDLE strOut]);
+        fprintf([WAITBAR_HANDLE WAITBAR_STRING]);
     end
 
     % Update carriage return
-    WAITBAR_HANDLE = repmat('\b',1,length(strOut)-1);
+    WAITBAR_HANDLE = repmat('\b',1,length(WAITBAR_STRING)-1);
 
-    if c == 100
+    if WAITBAR_PROGRESS == 100
         fprintf('\n');
     end
+elseif islogical(c)
```

Comment:
I am wondering if we shall not move `textprogressbar.m` to `tools/` - our version is highly customized already. @syarra ?